documentation/docs/libraries/nodejs/exchange_guide.md

```diff
@@ -0,0 +1,238 @@
+
+:::
+
+Since all `wallet.rs` bindings are based on core principles provided by the `wallet.rs` library, the outlined approach is very similar regardless of the programming language of your choice.
+
+### 1. Set up the Wallet Library
+First, you should install the components that are needed to use `wallet.rs` and the binding of your choice; it may vary a bit from language to language. In the case of the `NodeJs` binding, it is straightforward since it is distributed via the `npm` package manager. We also recommend you use `dotenv` for password management.
+
+You can read more about [backup and security in this guide](https://wiki.iota.org/introduction/guides/backup_security).
+
+```bash
+npm install @iota/wallet dotenv
+```
+
+#### 1.1 Generate a mnemonic
+
+<CodeBlock className="language-javascript">
+  {generate_mnemonic}
+</CodeBlock>
+
+Then, input your password to the `.env` file like this:
+
+```bash
+touch .env
+```
+
+```bash
+SH_PASSWORD="here is your super secure password"
+MNEMONIC="here is your super secure 24 word mnemonic, it's only needed here the first time"
+```
+
+Once you have everything needed to use the `wallet.rs` library, it is necessary to initialize the `AccountManager` instance with a secret manager(`Stronghold` by default) and client options.
+
+:::note
+
+Manage your password with the utmost care.
+
+:::
+
+By default the Stronghold file will be called `wallet.stronghold`. It will store the seed (derived from the mnemonic) that serves as a cryptographic key from which all accounts and related addresses are generated.
+
+One of the key principles behind the `stronghold` is that no one can get a seed out of it, so you should also backup the mnemonic (24 words) in a secure place, because there is no way to recover it from the `.stronghold` file. You deal with all the accounts purely via the `AccountManager` instance where all complexities are hidden under the hood and are dealt with securely.
+
+:::note
+
+Keep the `stronghold` password and the `stronghold` database on separate devices. See the [backup and security guide](https://wiki.iota.org/introduction/guides/backup_security) for more information.
+
+:::
+
+#### 1.2 Create an account
+
+Import the Wallet Library and create an account manager:
+
+<CodeBlock className="language-javascript">
+  {create_account}
+</CodeBlock>
+
+### 2. Create an Account For a User
+
+Once the backend storage is created, individual accounts for individual users can be created.
+
+The `Alias` can be whatever fits to the given use case and needs to be unique. The `Alias` is typically used to identify the given account later on. Each account is also represented by an `index` which is incremented (by 1) every time a new account is created. Any account can then be referred to via `index`, `alias`.
+
+Once an account has been created, you get an instance of it using `AccountManager.getAccount(accountId|alias)` or get all accounts with `AccountManager.getAccounts()`.
```

Comment:
```suggestion
You get an instance of any created account using [`AccountManager.getAccount(accountId|alias)`](../../reference/nodejs#getaccountaccountid) or get all accounts with [`AccountManager.getAccounts()`](../../reference/nodejs#getaccounts).
```

Reply:
Just in case, in the [current api ref](https://wiki.iota.org/wallet.rs/reference/nodejs#getaccountaccountid), there is one method to getAccount(ID) and another to getAccountByAlias(alias),

Reply:
Outdated api ref